### ISSUE TITLE
Backup friends, servers etc etc to a file in case of account loss

### ISSUE THREAD

**Issue by EnTechCore** [Vencord/plugin-requests#336]
### Plugin Description

Not like any of us trust Discord _not_ to remove our accounts one day.
Either that or I'm going to somehow lock myself out of my account. Just you wait.

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Antonio225t** [comment]
I agree... Infact Discord is starting to delete old accounts and I didn't want to lose my old DMs but this can be abused into spamming the API so... It was sadly rejected in https://github.com/Vencord/plugin-requests/issues/534

**Comment by Vendicated** [comment]
you can request your data package via privacy settings
